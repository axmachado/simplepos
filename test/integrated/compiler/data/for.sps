module repeat();

int x;

for ( x = 0; x < 5; x++) {
    waitkey();
}
