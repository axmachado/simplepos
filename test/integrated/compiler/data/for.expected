repeat ()
{
    for (x = 0, (x < 5), x++)
    {
        waitkey ()
    }
}
